pulseapi/profiles/test_views.py

```diff
@@ -504,3 +504,20 @@ def test_profile_categories(self):
         self.assertListEqual(categories['profile_types'], list(ProfileType.objects.values_list('value', flat=True)))
         self.assertListEqual(categories['program_types'], list(ProgramType.objects.values_list('value', flat=True)))
         self.assertListEqual(categories['program_years'], list(ProgramYear.objects.values_list('value', flat=True)))
+
+    def test_profile_view_is_active_filter(self):
+        """
+        Creating and saving an inactive profile, then preforming a GET
+        request to the profile endpoint with the '?is_active' filter set to
+        'True', then checking if any returned profiles are set to 'False'.
+        """
+        inactive_profile = BasicUserProfileFactory.create(is_active=False)
+        inactive_profile.save()
+
+        profile_url = reverse('profile_list')
+        url = ('{url}?is_active=True').format(url=profile_url)
```

Comment:
**Suggestion** I think it would be great to add another test where you use `url = ('{url}?is_active=False').format(url=profile_url)` and check that only the inactive profiles are returned. Because the "only active profiles" case is already the default, or am I off here?

Reply:
Added [test_profile_view_is_inactive_filter](https://github.com/mozilla/network-pulse-api/blob/ad7ee392ea97c16a6febe6894082ae538c1b7c86/pulseapi/profiles/test_views.py#L527) 👍